.travis.yml

```diff
@@ -10,7 +10,7 @@ envs:
 
 script:
   - tox
-  - ./node_modules/gulp/bin/gulp.js test
+  - make test-js
   - bash ./run_bokchoy_tests.sh
```

Comment:
Why isn't run_bokchoy_tests part of the `test` target?  Given that it's part of the test suite, it seems like we would want to run it locally as well.